src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/EmailConfigurationType.php

```diff
@@ -73,43 +80,60 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('send_emails_to', ChoiceType::class, [
+            ->add(self::FIELD_MAIL_EMAIL_MESSAGE, ChoiceType::class, [
```

Comment:
```suggestion
            ->add(self::FIELD_MAIL_EMAIL_MESSAGE, ChoiceType::class, [
```

Replace `self` by `static`